README.md

```diff
@@ -1,18 +1,16 @@
 
-This action will scan the file provided in the `file-with-substitutions` argument for Octopus variable substitution syntax `#{VariableName}`.  If it finds a matching variable in the `variables-file`, it will replace the template with the actual value.
-
-The variable value and keys can all be placed inside of one file.  The substitution has to be performed one file at a time though.
+This action will scan the file(s) provided in the `file-with-substitutions` argument for Octopus variable substitution syntax `#{VariableName}`.  If it finds a matching variable in the `variables-file` or in the environment variables, it will replace the template with the actual value.
 
 This is a container action so it will not work on Windows runners.
 
 ## Inputs
 
-| Parameter                 | Is Required | Description                                                                                                                                                      |
-| ------------------------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `variables-file`          | true        | The json file that holds the variables and values to be used in the substitution.<br/>Defined as JSON object, e.g. `{"Variable1":"value1","Variable2":"value2"}` |
-| `file-with-substitutions` | true        | The file that has substitutions to be performed.                                                                                                                 |
-| `output-file`             | true        | The file the substitution results should be saved to.  This is generally the same value provided in `file-with-substitutions`.                                   |
+| Parameter                  | Is Required | Description                                                                                                                                                                                                                                              |
+| -------------------------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `variables-file`           | false       | An optional yaml file containing variables to use in the substitution.                                                                                                                                                                                   |
+| `files-with-substitutions` | true        | A comma separated list of files that contains the octo tokens needing to be substituted.                                                                                                                                                                 |
```

Comment:
I don't know if you are tied to this wording but something like this might be a little smoother:
```suggestion
| `files-with-substitutions` | true        | A comma separated list of files with `#{variables}` that need substitution.                                                                                                                                                                 |
```